### PR TITLE
added an fr folder with sas statuts

### PR DESCRIPTION
**Pull request by albanlv**
Beware of the "if" part, I'm not sure I got it right :D

### CODE REVIEW

**Comment by zaiste** [comment]
@albanlv great work

File: views/fr/statuts_sas.html [hunk 1]
```diff
@@ -0,0 +1,456 @@
+<!DOCTYPE html>
+{{ @capital = @capital1.to_i + @capital2.to_i + @capital3.to_i + @capital4.to_i + @capital5.to_i }}
```
Comment (zaiste) [inline views/fr/statuts_sas.html:2]: was ist das? :) how about moving it to a `json` array with 5 elements ?

**Comment by albanlv** [comment]
sure, there is statuts_sas.html and statuts_sas.json, did I forgot something?

**Comment by zaiste** [comment]
@albanlv no, it's great I missed it

**Pull request merged**